QuantConnect.CoinbaseBrokerage.Tests/CoinbaseBrokerageDataQueueHandlerTests.cs

```diff
@@ -131,5 +140,121 @@ public void StreamsData(Symbol symbol, Resolution resolution)
 
             cancelationToken.Cancel();
         }
+
+        [Test]
+        public void SubscribeOnMultipleSymbols()
+        {
+            var configs = GetRandomSymbols();
```

Comment:
Try not to use random data for testing, it makes them irreproducible

Reply:
Remove random data.